composer.json

```diff
@@ -22,7 +22,7 @@
 		"nette/di": "^3.0",
 		"kdyby/strict-objects": "^2.0",
 		"monolog/monolog": "^2.0",
-		"tracy/tracy": "^3.0"
+		"tracy/tracy": "^2.7 || ^3.0"
```

Comment:
With the changes it's likely no longer compatible with 3.0. Which isn't a problem but you should disallow it here.

Reply:
You mean because of the removed string typehint? I think it still is. In php 7.2 and newer, type widening is allowed: https://wiki.php.net/rfc/parameter-no-type-variance

Reply:
tracy/tracy v3.0.x-dev was installed here, and the tests passed: https://travis-ci.org/github/Kdyby/Monolog/jobs/733206102

Reply:
okay